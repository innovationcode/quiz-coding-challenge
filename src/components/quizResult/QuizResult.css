.quiz-result {
  text-align: left;
  margin: 10% 20%;
  margin-top: 5%;
  margin-bottom: 2%;
  height: auto;
}
