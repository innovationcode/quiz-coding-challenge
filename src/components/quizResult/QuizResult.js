import React from 'react';
import './QuizResult.css';

function QuizResult({ score }) {
      return (
            <div className = "quiz-result"> 
                  <h2>QuizResult....</h2>
                  <h3>Your score is : {score}</h3>
            </div>
      )
}

export default QuizResult;
