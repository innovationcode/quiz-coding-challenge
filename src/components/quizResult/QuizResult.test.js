import React from 'react';
import { shallow } from 'enzyme';

import QuizResult from './QuizResult.js'; //component to be tested.

describe('Testing QuizResult component', () => {
      
      it('should render correctly in "debug" mode', () => {
            const wrapper = shallow(<QuizResult debug />);
            expect(wrapper).toMatchSnapshot();
      });

      

});