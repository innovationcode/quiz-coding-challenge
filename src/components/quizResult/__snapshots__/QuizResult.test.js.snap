// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing QuizResult component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
